deploy/examples/operator-openshift.yaml

```
@@ -658,6 +658,9 @@ spec:
             - name: ROOK_ENABLE_MACHINE_DISRUPTION_BUDGET
               value: "false"
 
+            - name: ROOK_DISABLE_ADMISSION_CONTROLLER
+              value: "true"
```

So we can change this default back to false, right?
```suggestion
              value: "false"
```